src/map/homunculus.cpp

```diff
@@ -644,21 +631,15 @@ int hom_evolution(struct homun_data *hd)
 	hom->luk += 10*rnd_value(min->luk, max->luk);
 	hom->intimacy = battle_config.homunculus_evo_intimacy_reset;
 
-	hom_calc_skilltree(hd);
```

Comment:
Did you intentionally remove this?

Reply:
Yes, official server does not recalculate the skilltree after evolution